tasks/skylab/FastqProcessing.wdl

```diff
@@ -0,0 +1,92 @@
+version 1.0
+
+task FastqProcessing {
+  input {
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    Array[File]? i1_fastq
+    File whitelist
+    String chemistry
+    String sample_id
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/fastq-process"
+    Int machine_mem_mb = 3850
+    Int cpu = 16   
+    #TODO decided cpu
+    # estimate that bam is approximately equal in size to fastq, add 20% buffer
+    Int disk = ceil(size(r1_fastq, "GiB")*3 + size(r2_fastq, "GiB")*3) + 500
+
+    Int preemptible = 3
+  }
+
+  # give the command 500MB of overhead
+  Int command_mem_mb = machine_mem_mb - 500
+
+  meta {
+    description: "Converts a fastq file into an unaligned bam file."
```

Comment:
Should the meta description also say that it attaches cell barcodes?

Reply:
changed